lang/cs/Org.Apache.REEF.Wake/Time/Runtime/RuntimeClock.cs

```diff
@@ -73,6 +74,11 @@ public sealed class RuntimeClock : IClock
             _runtimeStartHandler = runtimeStartHandler;
             _runtimeStopHandler = runtimeStopHandler;
             _idleHandler = idleHandler;
+            ++NumberOfInstantiations;
```

Comment:
This should stay, as it helps debugging things. And yes, it should be in the logger, not on `Console`.

We could also have a method in the Logger (or elsewhere) to log supposedly singleton instantiations. That way, we can reuse the code in other cases.